lib/data/editor/editor_core_info.dart

```diff
@@ -380,7 +384,7 @@ class EditorCoreInfo {
       'c': initialPageIndex,
     };
 
-    json['a'] = assets.map((Uint8List asset) => base64Encode(asset)).toList();
+    json['a'] = assets.map((Uint8List asset) => BsonBinary.from(asset.buffer.asUint8List())).toList();
```

Comment:
Is there any reason you went with `asset.buffer.asUint8List()` instead of just `asset`?

Reply:
No. I only did it like that because it worked like that for saving the strokes. The conversion was redundant.